### DISCUSSION TITLE
comoseria si no te pidieran es past ports

### DISCUSSION THREAD

**Discussion by Junior03216** [community/community#118592] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

opinen

**Comment by LiteBrite82** [comment]
Hi @Junior03216 ,

Thanks for joining us, we're glad you're here!

If you're looking for help with a general GitHub feature you might want to try asking in the main [GitHub Community](https://github.com/orgs/community/discussions/).

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one. [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) 

Best of luck!
